lib/linux_admin/hosts.rb

```diff
@@ -36,6 +38,10 @@ def update_entry(address, hostname, comment = nil)
       end
     end
 
+    def hostname=(name)
+      run!(cmd('hostnamectl'), :params => ['set-hostname', name])
```

Comment:
Yeah, that sounds good to me...We might want a higher level check for systemd instead of the individual commands, but this should work.

Also the params on the second one can be simplified to `:params => {:F => "/etc/hostname}`